ocaml/utils/targetint.ml

```diff
@@ -102,3 +102,11 @@ include (val
            | 64 -> (module Int64)
            | _ -> assert false
           ) : S)
+
+let size_in_bytes_as_targetint =
+  match size with 32 -> of_int32 4l | 64 -> of_int64 8L | _ -> assert false
+
+let to_uint64_exn t =
```

Comment:
Maybe this should be called `nonnegative_to_uint64_exn`, since I think it's going to raise on strictly negative numbers.

Please also add newlines at the end of the files `targetint.ml` and `targetint.mli`.

Reply:
Well spotted